torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6785,6 +6826,20 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestJit', 'test_variant_consistency_jit'),
            ),
            supports_out=False,),
+    OpInfo('nn.functional.layer_norm',
+           aten_name='layer_norm',
+           aliases=('layer_norm',),
+           ref=reference_layer_norm,
+           dtypes=floating_types_and(torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
+           supports_out=False,
+           decorators=[
+               DecorateInfo(
+                   toleranceOverride({torch.float32: tol(atol=1e-05, rtol=1e-03)}),
```

Comment:
What are the absolute values that are being compared? (I want to make sure the tolerance override isn't because of a bug somewhere)

Reply:
Thanks for the question, @zou3519. Here is the error detail: (let me know if this helps)

```python
AssertionError: False is not true : Tensors failed to compare as equal!With rtol=1.3e-06 and atol=1e-05, found 1 element(s) (out of 1) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 3.0279159545898438e-05 (2.5539915561676025 vs. 2.5540218353271484), which occurred at index 0
```

This was a failure with `test_reference_testing` test, so I think it's not a bug but might just be precision differences in NumPy vs PyTorch.

Reply:
Thanks for the numbers. This seems fine to me